.github/workflows/test-backend.yaml

```diff
@@ -42,7 +42,11 @@ jobs:
           VERSION: test
           RELEASEID: test
         run: |
-          python -m pytest --cov=app --cov-report=xml
+          coverage run --concurrency=thread,greenlet --source=app -m pytest
```

Comment:
silly question, does this run only the coverage or the tests as well?

Reply:
this runs the tests and measures the coverage during the test execution